exercises/webapp/src/components/NewsForm.vue

```diff
@@ -1,9 +1,11 @@
 <template>
-  <div id="news-input">
-    <label>
-      <input v-model="news_input" placeholder="edit me">
-    </label>
-    <button @click="createNewsItem">create</button>
+  <div>
+    <form @submit.prevent="createNewsItem" id="news-input">
+      <input id="news-input-txt" name="news-input-txt" v-model="news_input" placeholder="edit me"
+             aria-label="News Message">
+      <input type="submit" value="create" name="create-button" id="create-button" aria-label="Create">
+    </form>
+    <button @click="switchSortOrder" id="switch-order-button">switch sort order</button>
```

Comment:
This is odd: Why is a component which is called `NewsForm` responsible to change the order a parent component? :thinking: